src/main/java/at/hannibal2/skyhanni/mixins/transformers/renderer/MixinContributorRendererEntityLiving.java

```diff
@@ -17,12 +17,13 @@ public class MixinContributorRendererEntityLiving<T extends EntityLivingBase> {
     private final RendererLivingEntityHook skyHanni$hook = new RendererLivingEntityHook();
 
     @Redirect(method = "rotateCorpse", at = @At(value = "INVOKE", target = "Ljava/lang/String;equals(Ljava/lang/Object;)Z", ordinal = 0))
-    private boolean rotateCorpse(String displayName, Object v2, T bat, float p_77043_2_, float p_77043_3_, float partialTicks) {
-        return skyHanni$hook.rotateCorpse(displayName, bat);
+    private boolean checkName(String displayName, Object v2, T bat, float p_77043_2_, float p_77043_3_, float partialTicks) {
```

Comment:
can we keep using the original function name for consistency?

Reply:
I feel using this new function name is more descriptive of what part of the function is being mixed into, but if you really want I can revert the function name